docs/source/using/managing-minishift.adoc

```diff
@@ -7,17 +7,13 @@
 
 toc::[]
 
-
-The following sections describe different aspects of using Minishift and
-provide an overview of different components and services.
-
 [[managing-minishift-intro]]
 == Managing Minishift
 
-When you use Minishift, you interact with two components that are managed by Minishift:
+When you use Minishift, you interact with two components:
 
-- virtual machine (VM) created by Minishift
-- OpenShift cluster provisioned by Minishift within the VM
+- a virtual machine (VM) created by Minishift
+- the OpenShift cluster provisioned by Minishift within the VM
 
 The following sections contain information about managing the Minishift VM.
 For details about using Minishift to manage your local OpenShift cluster,
```

Comment:
Here at this line the link to `the Interacting with OpenShift` seems to be broken.